hknweb/academics/models.py

```diff
@@ -0,0 +1,3 @@
+from .models.course_surveys import Question, Rating, Survey
+from .models.icsr import ICSR
+from .models.logistics import Course, Department, Instructor, Semester

```

Comment:
Why do we need a models.py if there is a folder which contains all the models?

Reply:
My migrations originally errored out if I didn't import them explicitly here, but I just tried making the initial migration without the `models.py` file and everything worked out. My best guess is that I was having other errors before that could've caused Django to fall back on explicit definitions or maybe my filesystem structure was wrong, although not too sure. Removing the file now!